src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    coordinates: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coordinates.x}, y=${this.coordinates.y}`;
+    },
+    goRight(step = 1) {
```

Comment:
```suggestion
    get info() {
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
    },

    get location() {
      return `${name}: x=${this.coordinates.x}, y=${this.coordinates.y}`;
    },

    goRight(step = 1) {
```

Add blank lines between methods
Fix everywhere